common/predictive-text/worker/index.ts

```diff
@@ -0,0 +1,4 @@
+// Dummy module for getting environment setup.
+module.exports ={
+    hello: () => 'hello'
+}
```

Comment:
Problem:  this style won't work in the browser, and thus not for eventual integration with KMW/the mobile apps.  Fortunately, there is a relatively simple way to fix that:

https://www.matteoagosti.com/blog/2013/02/24/writing-javascript-modules-for-both-browser-and-node/

For my proposed fix, see https://github.com/keymanapp/keyman/blob/lmlayer-tdd-edit/common/predictive-text/worker/index.ts.